src/node_api.cc

```diff
@@ -570,8 +570,7 @@ class SetterCallbackWrapper
 
   /*virtual*/
   void SetReturnValue(napi_value value) override {
-    node::FatalError("napi_set_return_value",
-      "Cannot return a value from a setter callback.");
+    assert(false);  // Cannot return a value from a setter callback
```

Comment:
> Isn't `assert()` a no-op on a release build of node?

What matters is whether this code is compiled in debug mode or not, not what node was compiled as. But yes, in release mode this will be a no-op.

(Also, can you `#include <assert.h>` if you’re going with this?)

> What about simply forward-declaring `node::FatalError()` if internals are unavailable? 

Please don’t make code rely on things that Node has explicitly marked as internals, you’ll be sad when they go away (also, it wouldn’t work because node doesn’t `dllexport` them on Windows). If you *really* want access to `node::FatalError()`, a better option would be to open a PR that moves it to the public headers.